source/process/release-faq.rst

```diff
@@ -73,11 +73,11 @@ Frequently Asked Questions
  
 **Q: How is cherry-picking done?**
```

Comment:
Needs an answer.

Reply:
Can link to https://developers.mattermost.com/contribute/getting-started/branching/.

Reply:
Done. :)